cypress/e2e/room_board/uploadMultipleFilesInFolderInRoomBoard.feature

```diff
@@ -38,48 +38,64 @@ Feature: Room Board - Upload multiple files in folder in board
         When I click on the icon Close on the fullscreen image
         Then I see page Folder content for '<folder_name>'
 
+        #editor opens audio in audio player
+        When I click on the name of file '<audio_file_name>' in file list
+        Then I see audio player
+        When I click on icon Download in the fullscreen image
+        Then file '<audio_file_name>' is saved in folder downloads
+        When I click on the icon Close on the fullscreen image
+        Then I see page Folder content for '<folder_name>'
+
+        #editor opens video in video player
+        # When I click on the name of file '<video_file_name>' in file list
+        # Then I see video player
+        # When I click on icon Download in the fullscreen image
+        # Then file '<video_file_name>' is saved in folder downloads
+        # When I click on the icon Close on the fullscreen image
+        # Then I see page Folder content for '<folder_name>'
+
         # editor checks file / multiple files
-        When I check the checkbox of file '<file_name_2>'
+        When I check the checkbox of file '<audio_file_name>'
         Then I see displayed number of checked files is '1'
         Then I see fab button Action at the top of the list
-        When I uncheck the checkbox of file '<file_name_2>'
+        When I uncheck the checkbox of file '<audio_file_name>'
         Then I do not see fab button Action at the top of the list
-        When I check the checkbox of file '<file_name>'
-        When I check the checkbox of file '<file_name_2>'
+        When I check the checkbox of file '<video_file_name>'
+        When I check the checkbox of file '<audio_file_name>'
         When I check the checkbox of file '<image_file_name>'
         Then I see displayed number of checked files is '3'
         Then I see state of table header checkbox is 'checked'
-        When I uncheck the checkbox of file '<file_name_2>'
+        When I uncheck the checkbox of file '<audio_file_name>'
         Then I see state of table header checkbox is 'mixed'
         Then I see displayed number of checked files is '2'
         Then I see fab button Action at the top of the list
 
         # editor checks / unchecks all files
         When I check the checkbox in the table header for all elements
-        Then I see checkboxes of files '<file_name>, <file_name_2>, <image_file_name>' are checked
+        Then I see checkboxes of files '<video_file_name>, <audio_file_name>, <image_file_name>' are checked
         When I uncheck the checkbox in the table header for all elements
-        Then I see checkboxes of files '<file_name>, <file_name_2>, <image_file_name>' are unchecked
+        Then I see checkboxes of files '<video_file_name>, <audio_file_name>, <image_file_name>' are unchecked
 
         # editor searches for files and uses check all checkbox
         When I enter '<search_request_2>' to the table search field
```

Comment:
`<search_request_2>`  -> there is only one `search_request` in the example table, so adapt it